common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -321,6 +321,11 @@
          */
         public static final String SCOPE = "scope";
 
+        /**
+         * Scope value used in acquiring tokens with saml assertion.
+         */
+        public static final String MSID_OAUTH2_SCOPE_OPENID_VALUE = "openid";
```

Comment:
Oops. Looks like we already have a constant for this
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/6dda4f95ca792b6c6f77a69347d56693dc8e75e1/common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java#L1233

Reply:
> Oops. Looks like we already have a constant for this

I was following the constants added in iOS for supporting the assertion feature should have checked if this one was present. I have removed this constant and made changes in the ADAL SDK too

Reply:
Hey @iambmelt, I have made the requested changes and added a new commit. Hoping everything else is fine and resolving the conversation